apps/api/src/domain/models/vocabulary-list/queries/vocabulary-list-query-repository.interface.ts

```diff
@@ -17,7 +18,8 @@ export interface IVocabularyListQueryRepository
     extends IPublishable,
         IAccessible,
         IQueryRepositoryForTaggable,
-        IQueryRepositoryForAnnotatable {
+        IQueryRepositoryForAnnotatable,
```

Comment:
I wanna look at introducing a `IResourceQueryRepository`. There is definitely no risk in having a base interface. And this would be a helpful contract if we ever implement the repositories with a second db.